### PR TITLE
feat: Add Playwright Test

### PR DESCRIPTION
**Pull request by nitya**
1. Config file specifies baseURL for hosted server
2. Test specification file configured for 2 simple tests (title, #items)
3. npm script updated to run Playwright test by default

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    trace: 'on-first-retry',
+
+    /* See: https://playwright.dev/docs/test-advanced */
+    baseURL: 'http://localhost:4280/home',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
```
Comment (johnpapa) [inline playwright.config.ts:36]: will this one also cover Edge as its chromium based? or do we need the branded ones for that?

File: tests/example.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+test.beforeEach(async ({ page }) => {
+  await page.goto('/');
+});
+
+test('Homepage has List with specified title', async ({ page }) => {
+  const listTitle=page.locator('.content-title-group h2');
+  await expect(listTitle).toHaveText("My List");
+});
+
+test('Homepage has List with 3 items', async ({ page }) => {
```
Comment (johnpapa) [inline tests/example.spec.ts]: can you change this to have at least 3 items? part of the demo adds one.

File: tests/example.spec.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test';
```
Comment (johnpapa) [inline tests/example.spec.ts]: Can you rename the file to `home.spec.ts` of `list.spec.ts` , so it doesnt appear to be a sample?

File: playwright.config.ts (hunk 1 above)
Comment (nitya) [reply playwright.config.ts:36]: Default Chromium option runs against latest version of Chromium and is recommended (valid for Chrome and Edge future proofing) but if required to run against specific versions (or stock versions) then configuration can be updated
https://playwright.dev/docs/browsers#google-chrome--microsoft-edge

**Pull request merged**